.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Not related to your PR, but can you add DS store too?

Reply:
Added here: [2e3170a](https://github.com/sukritsangvong/wp-browser-extension/pull/5/commits/2e3170ae22e82b4d51eb41899f446cf721689e25)